pyreleaseplugin/release.py

```diff
@@ -168,11 +169,18 @@ def build():
         raise RuntimeError("Error building wheel")
 
 
-def publish_to_pypi():
+def publish_to_pypi(repository):
     """
     Publish the distribution to our local PyPi.
+    :param repository: the repository to be used by the ReleaseCommand for the artifact upload. If ommitted, repository will be defaulted to PyPi
     """
-    code = Popen(["twine", "upload", "dist/*", "-r", "artifactory"]).wait()
+    cmd = None
```

Comment:
Nit: it would be slightly simpler as follows:

```python
if repository is None:
    repository = "pypy"

cmd = ["twine", "upload", "dist/*", "--repository", repository]
```